reference/test/matrix/csr_kernels.cpp

```diff
@@ -230,6 +231,20 @@ TEST_F(Csr, MovesToCoo)
     assert_equal_to_mtx(coo_mtx.get());
 }
 
+TEST_F(Csr, ConvertsToEll)
+{
+    auto ell_mtx = gko::matrix::Ell<>::create(mtx->get_executor());
+    auto dense_mtx = gko::matrix::Dense<>::create(mtx->get_executor());
+    auto ref_dense_mtx = gko::matrix::Dense<>::create(mtx->get_executor());
+
+    mtx->convert_to(ell_mtx.get());
+    ell_mtx->convert_to(dense_mtx.get());
+
+    mtx->convert_to(ref_dense_mtx.get());
+
+    GKO_ASSERT_MTX_NEAR(dense_mtx.get(), ref_dense_mtx.get(), 0.0);
```

Comment:
I don't think this test is enough, you also need one test to check if the values are set appropriately by hand (when just converting from CSR to ELL).